nemo_skills/evaluation/evaluator/bird.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import json
+import logging
+import os
+import re
+from pathlib import Path
+
+from func_timeout import FunctionTimedOut, func_timeout
+
+from nemo_skills.dataset.birdbench.evaluation import execute_sql
+from nemo_skills.evaluation.evaluator.base import BaseEvaluator, BaseEvaluatorConfig
+from nemo_skills.file_utils import jdump
+from nemo_skills.utils import nested_dataclass
+
+
+@nested_dataclass(kw_only=True)
+class BirdEvaluatorConfig(BaseEvaluatorConfig):
+    timeout: int = 30
+
+    # Answer format can be "BOXED", "CODEBLOCK", or "USE_REGEX", the last of
+    # which uses the given regex in the extraction_regex arg.
+    answer_format: str = "CODEBLOCK"
+    extraction_regex: str | None = None
+    regex_dotall: bool = False
+
+    db_path: str = ""
```

Comment:
I think we can always treat this relative to the dataset path, right? So we shouldn't have to specify it in the config--see our logic for getting the dataset path here: https://github.com/NVIDIA-NeMo/Skills/blob/main/nemo_skills/dataset/prepare.py#L35-L52

Reply:
Also, I think making this empty rather than having no default will lead to unexpected errors if someone doesn't know to update db_path

Reply:
Referring to the example I mentioned in the other comment where the database directory can be located in different places depending on where the code is executed, in the cluster execution case I believe it _should_ actually be specifically decoupled as it should be uploaded separately first--it's a somewhat large directory that shouldn't be copied to the cluster for each run.

> Also, I think making this empty rather than having no default will lead to unexpected errors if someone doesn't know to update db_path

Related to other comment--I can add a check for empty `db_path` to be more informative if that's helpful.

Reply:
if you leave it as:
```
    db_path: str
```
won't hydra enforce it is specified? Rather than have to implement the check

Reply:
Oh, I didn't know that! I'll update that and remove the manual check.